tests/integration/plugins/test_python_plugin.py

```diff
@@ -111,9 +111,13 @@ def test_python_plugin_ubuntu(base, tmp_path, run_lifecycle):
     assert list(bin_dir.glob("python*")) == []
 
     # Check the shebang in the "hello" script
-    expected_shebang = "#!/bin/python3"
+    # In test env this could be replaced with sh that exec python by the venv
+    expected_shebang = ("#!/bin/python3", "#!/bin/sh")
```

Comment:
I'm a bit confused here, how the starbase migration affected this test?

Reply:
This one is definitely a problem: the `#!/bin/python3` shebang should be set [by the Python plugin](https://github.com/canonical/rockcraft/blob/main/rockcraft/plugins/python_plugin.py#L87) and the purpose of the test is to verify this. `#!/bin/sh` is not an acceptable value, we need to look into this more

Reply:
```
#!/bin/sh
'''exec' /run/user/1000/tox_tmp/integration-py3.10/pytest-of-ubuntu/pytest-0/test_python_plugin_ubuntu_ubun1/parts/my-part/install/bin/python3 "$0" "$@"
' '''
# -*- coding: utf-8 -*-
import re
import sys
from hello.__main__ import main
if __name__ == '__main__':
    sys.argv[0] = re.sub(r'(-script\.pyw|\.exe)?$', '', sys.argv[0])
    sys.exit(main())
```

The file become like this if run under `tox`.

Reply:
Right I understand that, but we have to figure out a way around that because as it is the test is not checking what it should be checking

Reply:
I don't see any configuration can disable that. We may have to get a separate test for it without run in tox.

Reply:
Well, I can create a spread test for that, but it is not possible to do in tox.

Reply:
Added